server/api_test.go

```diff
@@ -1 +1,60 @@
 package server
+
+import (
+	"context"
+	"testing"
+
+	"golang.org/x/crypto/bcrypt"
+
+	"github.com/coreos/dex/api"
+	"github.com/coreos/dex/storage/memory"
+)
+
+var s = memory.New()
```

Comment:
So just have one test that
1. Creates a password.
2. Updates the password.
3. Deletes the password